### PR TITLE
add target iflight f745 aio

### PR DESCRIPTION
**Pull request by Kaiowarez**
Thank you for contributing to EmuFlight!

### CODE REVIEW

**Review by frizzle101101 — COMMENTED** [review]
not sure but dont think the betaflightf4 change is to be included...

**Comment by nerdCopter** [comment]
can't accept this with Betaflight target modded in same branch

**Review by frizzle101101 — CHANGES_REQUESTED** [review]
i think you are missing #define USE_LED_STRIP

**Comment by frizzle101101** [comment]
> i think you are missing #define USE_LED_STRIP, if that is realted to @Mcrubens comment on #479

i realize the led pin was not broken out on the v1 but on the v1.2 it is, so please add 
`#define USE_LED_STRIP
#define WS2811_PIN                      PD12`

also id sugest removing/disabling/comenting out, the unused/not-broken-out uarts like UART_8, UART_6, UART_5, only 1,2,4 and 7 are broken out

**Comment by frizzle101101** [comment]
I hope this helps with your next target adds, i sure learned somthing about Legacy targets lol

**Comment by nerdCopter** [comment]
Thank you @frizzle101101 !

@Kaiowarez , there is multiple comments(info) that needs attention above, thank you good sir!

File: src/main/target/BETAFLIGHTF4/target.h [hunk 1]
```diff
@@ -118,10 +117,10 @@
...
 
 #define USE_I2C
-#define USE_I2C_DEVICE_2
-#define I2C2_SCL                NONE // PB10, UART3_TX
-#define I2C2_SDA                NONE // PB11, UART3_RX
-#define I2C_DEVICE              (I2CDEV_2)
+#define USE_I2C_DEVICE_1
+#define I2C1_SCL                NONE // PB10, UART3_TX
+#define I2C1_SDA                NONE // PB11, UART3_RX
+#define I2C_DEVICE              (I2CDEV_1)
```
Comment (frizzle101101) [inline src/main/target/BETAFLIGHTF4/target.h:123]: remove changes

File: src/main/target/BETAFLIGHTF4/target.h [hunk 2]
```diff
@@ -74,7 +74,6 @@
 #define USE_USB_DETECT
 #define USB_DETECT_PIN          PC5
 
-
```
Comment (frizzle101101) [inline src/main/target/BETAFLIGHTF4/target.h:77]: remove changes

**Review by frizzle101101 — COMMENTED** [review]
here you go these are clear instructions to get this done asap by anyone who can read

**Comment by nerdCopter** [comment]
* thank you @frizzle101101 .  i will take a look and attempt this.
* yesterday i made a new PR with both your and kaio's authorship in my commits.  
* i will see about adding these things to #682 .
* regarding DPS310, @JulioCesarMatias was adding DPS310 in PR #578 , but the testers stopped responding.

**Comment by nerdCopter** [comment]
Superseded by #686

**Pull request closed without merging**